### DISCUSSION TITLE
Access denied for user 'root'@'localhost' (using password: NO)

### DISCUSSION THREAD

**Discussion by rahmaazs** [leokhoa/laragon#451] [Q&A]
I got error when i try to create a new Wordpress project in Laragon.

The error says like this:
![image](https://user-images.githubusercontent.com/113582506/227465882-ebbb4c92-292d-4318-826a-7cdcee2e99c3.png)

Does anyone know how I can fix this?

**Comment by Pen-y-Fan** [comment]
You have probably set the password for MySQL. Quick app will only work with Laragon default settings, no password.

**Reply by rahmaazs** [reply]
as i think, i've never set any password before. that's why it says "password: NO"

**Reply by Pen-y-Fan** [reply]
To check MySQL, in Laragon Start All, and click Terminal. From the terminal type:

```sh
mysql -u root
```

If you have a password you will see the access denied message.

You should see:

```text
Welcome to the MySQL monitor.  Commands end with ; or \g.
Your MySQL connection id is 8
Server version: 8.0.30 MySQL Community Server - GPL

Copyright (c) 2000, 2022, Oracle and/or its affiliates.

Oracle is a registered trademark of Oracle Corporation and/or its
affiliates. Other names may be trademarks of their respective
owners.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

mysql>
```

**Comment by gravitas-navi** [comment]
Believe it or not, I literally just ran into this a few minutes ago. At least from what I see, you can use quick app with a set password, but you have to edit your MySQL my.ini file under MySQL -> my.ini. You'll see on the first line a commented #password=your_password, and then you just uncomment and replace the placeholder your_password with whatever your current root password is as a string. After that, quick app will just read off of that and work how it did before.

I almost forgot you'll also have to edit your wp-config.php file inside whichever directory as well to include your current `DB_PASSWORD` (your root or whichever database user you plan to have as a login). Otherwise you'll run into a `Error establishing a database connection` 404 page. So it's not as much of a "quick app" once you change your root/user password, but still.

I just created two test sites after the fact, so maybe try that first.

**Reply by rahmaazs** [reply]
i've tried to edit the my.ini file and wp-config.php file just like what you said before, but it still doesn't work :(

**Reply by QueenChangge** [reply]
thank u, too much possibilities, i have done with this

**Reply by Kitilyes** [reply]
worked like a charm

**Comment by bastianGd** [comment]
To fix it, go to laragon\etc\apps\phpMyAdmin, then find config.inc.php, open it and find // $cfg['Servers'][$i]['AllowNoPassword'] = TRUE;
if that line was commented, remove '//' or if its on false set it on true. 
I did this and still having problems, so y changed the port, 80 to 90 and 3306 to 3307. It works for me

**Comment by Penworks** [comment]
Adding a comment in case helpful to other readers. Go to Laragon icon, right click for Menu/Mysql/Change root password (or generate random root password). After that you can deal with if you want no password or not. On localhost no password is probably ok but never use no password on live projects!

**Comment by RahmatPrakoso276** [comment]
i fix the same problem u facing

To check if any application is using port 3306, follow these steps:

1. Open the terminal and enter the following command:
netstat -ano | findstr :3306

above command will search for all connections and processes using port 3306.
example:
  TCP    [::]:3306              [::]:0                 LISTENING       1234

2. If you find a process using port 3306, you can identify it by its Process ID (PID) with the following command:
tasklist | findstr 1234

Replace 1234 with the PID shown in the previous output.

3. To stop the application using port 3306, run this command:
taskkill /PID 1234 /F

Replace 1234 with the actual PID you found.

**Comment by MTiberiu** [comment]
I had the same problem, the fix for me was to close the discord app

**Comment by K4nes** [comment]
I just installed the latest version 8.0.41, and it works perfectly
